unit_tests/src/test_logical_locking.cpp

```diff
@@ -175,11 +212,26 @@ TEST_CASE("open and close", "[write_lock]")
         // open existing, currently locked replica
         fail_cond_input[RESC_HIER_STR_KW] = other_replica_resc;
         REQUIRE(LOCKED_DATA_OBJECT_ACCESS == rcDataObjOpen(&new_comm, &fail_open_inp));
+
+        // ...and now, with a replica token
+        fail_cond_input[REPLICA_TOKEN_KW] = token;
+
+        // open existing, currently opened replica
+        {
+            fail_cond_input[RESC_HIER_STR_KW] = opened_replica_resc;
+            const auto fd2 = rcDataObjOpen(&new_comm, &fail_open_inp);
+            REQUIRE(fd2 > 2);
+            REQUIRE(close_replica(new_comm, fd2) >= 0);
+        }
+
+        // open existing, currently locked replica
+        fail_cond_input[RESC_HIER_STR_KW] = other_replica_resc;
+        REQUIRE(LOCKED_DATA_OBJECT_ACCESS == rcDataObjOpen(&new_comm, &fail_open_inp));
     }
 
-    SECTION("open for create")
+    SECTION("open_for_create")
     {
-        fail_open_inp.openFlags = O_CREAT | O_WRONLY | O_TRUNC;
+        fail_open_inp.openFlags = O_CREAT | O_WRONLY;
```

Comment:
just curious, please explain why no more trunc?

Reply:
I added a new piece to this test where the replica token DOES allow an additional open to succeed. If `O_TRUNC` is present, the file will be truncated. In this test, the file has contents and the purpose of this test is for checking only to see if the open would succeed. Can add back, but it wasn't really necessary to begin with and, if it is added back, I think there should be some additional assertions around the file's contents.

Reply:
That's fine fine - just seemed an important change without your usual storytelling.   Resolving.